front-end/src/components/UserHome.tsx

```diff
@@ -28,21 +22,25 @@ const UserHome = () => {
       console.log(res);
```

Comment:
remove this console. All console.log should be remove for prod ready.

Reply:
Done